src/docker-in-docker/install.sh

```diff
@@ -18,8 +18,8 @@ USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
 INSTALL_DOCKER_BUILDX="${INSTALLDOCKERBUILDX:-"true"}"
 INSTALL_DOCKER_COMPOSE_SWITCH="${INSTALLDOCKERCOMPOSESWITCH:-"true"}"
 MICROSOFT_GPG_KEYS_URI="https://packages.microsoft.com/keys/microsoft.asc"
```

Comment:
Can we bump minor version for the Feature? Thanks!

Reply:
https://github.com/devcontainers/features/pull/971/commits/6b6ad16487edbe685c598ed57c0ada36995648df